MCPForUnity/UnityMcpServer~/src/tools/manage_gameobject.py

```
@@ -51,20 +51,66 @@ def manage_gameobject(
+                return True
+            if v in ("false", "0", "no", "off"):
+                return False
+        return bool(value)
+
+    def _coerce_vec(value, default=None):
+        if value is None:
+            return default
+        if isinstance(value, list) and len(value) == 3:
+            try:
+                return [float(value[0]), float(value[1]), float(value[2])]
+            except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)